php_quickhash.h

```diff
@@ -37,11 +37,18 @@ PHP_MINFO_FUNCTION(quickhash);
 
 ZEND_BEGIN_MODULE_GLOBALS(quickhash)
 	/* empty */
-ZEND_END_MODULE_GLOBALS(quickhash) 
+ZEND_END_MODULE_GLOBALS(quickhash)
 
 #ifdef ZTS
+#if PHP_VERSION_ID < 70000
 # define QUICKHASH_G(v) TSRMG(quickhash_globals_id, zend_quickhash_globals *, v)
 #else
+#if defined(COMPILE_DL_QUICKHASH)
+	ZEND_TSRMLS_CACHE_EXTERN();
```

Comment:
Instead of using ZEND_ in the name, it probably should be the one without, aka: ```TSRMLS_CACHE_EXTERN```.